packages/console/src/pages/Connectors/components/ConnectorRow/index.tsx

```diff
@@ -67,14 +68,15 @@ const ConnectorRow = ({ type, connectors, onClickSetup }: Props) => {
       </td>
       <td>{connectorTypeColumn}</td>
       <td>
-        {inUse !== undefined && (
-          <Status status={inUse ? 'enabled' : 'disabled'}>
-            {t('connectors.connector_status', {
-              context: inUse ? 'in_use' : 'not_in_use',
-            })}
-          </Status>
-        )}
-        {inUse === undefined && !connectors[0] && '-'}
+        {conditional(
+          firstConnector && (
```

Comment:
Why does only the first connector need this status?

Reply:
For multi-platform cases, there will be more than one connector.